Extensions/Ansible/Src/Tasks/Ansible/task.json

```diff
@@ -10,13 +10,14 @@
         "Release"
     ],
 	"runsOn": [
-        "Agent"
+        "Agent",
+        "DeploymentGroup"
     ],
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
         "Minor": 0,
-        "Patch": 4
+        "Patch": 5
     },

```

Comment:
You need to update the extension version as well

Reply:
Done